### ISSUE TITLE
Can't launch applications

### ISSUE THREAD

**Issue by madjpm** [mikelolasagasti/revelation#80]
## Overview
I can't open any external application like using gnome-terminal (fro SSH or TELNET connexions).

Revelation - 0.5.4

[NOTE]: # ( Pick choices based on your environment )
Operating System: Mint 20
Desktop Env: Mate

**Comment by mikelolasagasti** [comment]
Can you try changing the shell entry in `Preferences` -> `Goto commands` to this:

```gnome-terminal -- ssh -l %u %h```

**Comment by madjpm** [comment]
> Are you using the flatpak version?
> 
> Can you try changing the shell entry in `Preferences` -> `Goto commands` to this:
> 
> `gnome-terminal -- ssh -l %u %h`

It does not work. Still nothing.

**Comment by mikelolasagasti** [comment]
Are you using the Flatpak version? You can check using `flatpak ps` and check if revelation shows in the output.

And if so, as Flatpak version runs in a sandbox I think it won't work with current permissions, at least based on my tests.

**Comment by madjpm** [comment]
Yes I'm using the flatpak version, can't find a binary version of revelation for linux mint 20
